### PR TITLE
Modernize embulk-parser-json with v0.11-ready API/SPI

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: embulk-parser-json/build.gradle [hunk 1]
```diff
@@ -62,6 +70,88 @@ javadocJar {
...
+
+                developers {
+                    developer {
+                        name = "Muga Nishizawa"
+                        email = "muga.nishizawa@gmail.com"
+                    }
+                    developer {
+                        name = "Masatoshi Shimada"
+                        email = "masatoshi.shimada@every.tv"
+                    }
```
Comment (dmikurube) [inline embulk-parser-json/build.gradle:110]: @smdmts We're going to separate `embulk-standards` plugins as individual plugins, then to release them as independent pure-Java Embulk plugin JARs into Maven Central. In this `embulk-parser-json`, we wanted to include you as a developer from the pull request #651. What do you think?

**Comment by dmikurube** [comment]
`embulk-util-file` will be updated to `0.1.2` in another PR after this is merged to `master`. (#1410 needed to be on this branch to use `0.1.2`, but this PR was developed before #1410. Rebasing everything would be hard...)

File: embulk-parser-json/build.gradle (hunk 1 above)
Comment (smdmts) [reply embulk-parser-json/build.gradle:110]: @dmikurube Sorry for delay replying. I approve include my name and appreciate it. Thank you.

**Comment by dmikurube** [comment]
`embulk-util-file:0.1.3` would work. Updated.

**Review by huylenq — APPROVED** [review]
LGTM

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**